utils/txHandler.ts

```diff
@@ -76,6 +76,15 @@ export const getNextAssetId = async () => {
   return new BN(asset_id.toString());
 };
 
+export const getUserBalanceOfToken = async (tokenId: BN, account: User) => {
```

Comment:
Optional: 
This function is not TransactionHandler related, looks more like a utils or related to Asset. plz move this to keep the functions more or less organized.

Reply:
Added to Utils.ts